### ISSUE TITLE
Eve Serivices/Characteristiken

### ISSUE THREAD

**Issue by hobbyquaker** [HM-RedMatic/RedMatic-HomeKit#114]
Leistungsmessung, Diagramme, ...

**Comment by superpower10** [comment]
Ja, dass fehlt mir im Gegensatz zur Homebridge Homematic Variante auch sehr.

Dort kann man z.B. auch mal nachschauen, wann denn nun die Tiefsttemperatur erreicht wurde

![IMG_9A9B508A7660-1](https://user-images.githubusercontent.com/20533966/56467102-9b5cfa80-641a-11e9-95a2-108f61582d70.jpeg)

**Comment by rost2802** [comment]
Ja das wäre was tolles!

**Comment by udochrist** [comment]
Gibts in dem Thema denn schon Fortschritt / weitere Überlegungen?
Wenn ich das richtig sehe ists bisher ein separater branch der vor ewiger zeit stehengeblieben ist.

**Comment by hobbyquaker** [comment]
RedMatic-HomeKit **4.0.0** ist seit heute veröffentlicht – Installation in RedMatic 9 über den Node-RED-Palettenmanager (Suche nach `redmatic-homekit`), Release-Notes: https://github.com/rdmtc/RedMatic-HomeKit/releases/tag/v4.0.0.

Eve-Characteristics (Leistung, Energie, Verlauf) sind für 4.0.0 nicht drin; das Thema bleibt offen und sammelt die Wünsche aus #227 und #379.

---
*Written by Claude Fable on behalf of hobbyquaker.*
